core/src/testimony_recorder.cpp

```diff
@@ -70,3 +76,13 @@ void AOClient::clearTestimony()
     AreaData* area = server->areas[current_area];
     area->clearTestimony();
 }
+
+bool AOClient::checkTestimonySymbols(QString message)
+{
+    if (message.contains('>') || message.contains('<')) {
+        sendServerMessage("Unable to add statements containing '>' or '<'.");
+        return true;
+    }
+    else
+        return false;
+}
```

Comment:
```suggestion
bool AOClient::checkTestimonySymbols(const QString& message) const
{
    if (message.contains('>') || message.contains('<')) {
        sendServerMessage("Unable to add statements containing '>' or '<'.");
        return true;
    }
        
    return false;
}
```

A `return` will quit early, so you can cheese it this way.
Also added the `const` and reference stuff here, too.